Fastfile

```diff
@@ -75,115 +73,113 @@ end
 
 # --------- Custom functions --------- #
 
-def _setup()
+def _setup
   ENV['SCAN_SCHEME'] = ENV['GYM_SCHEME']
-  if ENV['SCAN_DEVICE'] == nil
-    ENV['SCAN_DEVICE'] = "iPhone 6 (9.3)"
-  end
-  if is_ci && ENV['TAB_XCODE_PATH'] != nil
-    xcode_select(ENV['TAB_XCODE_PATH'])
-  end
-  if ENV['TAB_UI_TEST_SCHEME'] != nil && ENV['TAB_OUTPUT_TYPES'] == nil
-    ENV['TAB_REPORT_FORMATS'] = "html"
-  end
-  if ENV['TAB_UI_TEST_SCHEME'] != nil && ENV['TAB_UI_TEST_DEVICES'] == nil
-    ENV['TAB_UI_TEST_DEVICES'] = "iPhone 8"
+  ENV['SCAN_DEVICE'] = 'iPhone 6 (9.3)' if ENV['SCAN_DEVICE'].nil?
```

Comment:
Maybe `ENV['SCAN_DEVICE'] ||= 'iPhone 6 (9.3)'` ?

Reply:
I actually prefer the way it is, it's easier to read for people who aren't super familiar with Ruby which makes it more accessible to contribute

Reply:
Yeah, nice man - I forgot about that beauty